pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java

```diff
@@ -145,16 +148,19 @@ public SuccessResponse deleteSchema(
   @Authenticate(AccessType.UPDATE)
   @ApiOperation(value = "Update a schema", notes = "Updates a schema")
   @ApiResponses(value = {
-      @ApiResponse(code = 200, message = "Successfully updated schema"),
-      @ApiResponse(code = 404, message = "Schema not found"),
-      @ApiResponse(code = 400, message = "Missing or invalid request body"),
-      @ApiResponse(code = 500, message = "Internal error")
+      @ApiResponse(code = 200, message = "Successfully updated schema"), @ApiResponse(code = 404, message = "Schema "
```

Comment:
Don't change this format (if you apply the latest checkstyle rules, it should not reformat this)

Reply:
Ack.
Even with latest codestyle.xml, intellij seems to be doing this when I auto format the file. Have fixed these. Will look into the codestyle